packages/muze-legend/src/legend/legend-helper.js

```diff
@@ -27,16 +27,98 @@ export const getScaleInfo = (scale) => {
 };
 
 /**
- *
- *
- * @param {*} domain
- * @param {*} steps
- *
+ * @param  {} domain
+ * @param  {} domainForLegend
+ * @param  {} context
  */
-export const getInterpolatedData = (domain, steps) => {
+export const getInterpolatedArrayData = (domain, domainForLegend, context) => {
+    const { align } = context.config();
+    const availableSpace = context.measurement();
```

Comment:
First let statments. No let statements to be used after declaration of const and/or computation statements in a single blocked(blocked lets are allowed).
Const can be declared after computation.